doc/Learn/Maui/ThirdParty-GrialKit.md

```diff
@@ -28,21 +31,56 @@ The last thing we'll need from the zip file is the nuget.config file. This file
 
 ## Getting Started
 
-1. Create a new application using the `unoapp` template, enabling .NET MAUI Embedding. In this case, we're going to use the Blank template (`-preset blank`) and include .NET MAUI Embedding support (`-maui`).
+### [Visual Studio](#tab/vs)
```

Comment:
@Kunal22shah @agneszitte I think we should reduce the "Getting Started" section:
- For Visual Studio, run the Uno Platform Template Wizard and enable .NET Maui Embedding (removing other platforms isn't required)
- For CLI, run "dotnet new unoapp maui -o GrialKitApp"
The focus on this doc should be anything related to referencing GrialKit